portfolio/src/main/webapp/js/script.js

```diff
@@ -12,6 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Define the companies' descriptions table
+const abouts = [
+    'Google LLC is an American multinational technology company that specializes in Internet-related services and products, which include online advertising technologies, a search engine, cloud computing, software, and hardware.',
+    'Koç University is a non-profit private university in Istanbul, Turkey. It started education in temporary buildings in İstinye in 1993, and moved to its current Rumelifeneri campus near Sarıyer in 2000.\
+     Koç University is ranked highest in Turkey according to the 2018 Times Higher Education World University Rankings.',
+    'The Polytechnique Montréal is an engineering school affiliated with the Université de Montréal in Montreal, Quebec, Canada. It ranks first in Canada for the scope of its engineering research.\
+     It is occasionally referred to as Montreal Polytechnic, although in Quebec English its French name is more commonly used.',
+    'Youth Fusion is an award-winning charity that contributes to perseverance, employability and civic engagement of youth by implementing\
+     innovative experiential learning projects that create ongoing links between school systems and the community.',
+    'The Algiers STEAM Center strengthens the innovation, critical thinking, and communication skills of the Algerian workforce via direct training of youth,\
+     targeted teacher training, and community education forums and events.',
+];
+
+// Define my responsabilities during my experiences
+const responsabilities = [
```

Comment:
Google has an 80 column length limit on its JS style guide (https://google.github.io/styleguide/jsguide.html#formatting-column-limit). Please break up these string constants using '+' so that each line has at most 80 columns.

Reply:
Okay! Thanks